bin/run-tests-in-docker.sh

```diff
@@ -22,7 +22,6 @@ docker build --rm -t exercism/test-runner .
 docker run \
     --rm \
     --network none \
-    --read-only \
```

Comment:
Just to make you aware. I am rather sure that you aren't allowed to do this. I think when they run in the containers are they running in read-only environments, I will ask Jeremy or Erik but I think doing something like that aint allowed.

Reply:
Hm ... that might be a bit of a problem :( Godot needs to create some extra folders (at least in ~/.local, I think there might be more) where the configuration of each individual project is stored. Without a permission to modify the filesystem, Godot fails to start. I might be able to create those folders manually in Dockerfile, I'll try to do that later.

Reply:
Well, we don't _yet_ enforce read only, but we might at some point in the future. The `/tmp` folder _is_ writeable, as well as the `output` directory passed in as the third argument.

Reply:
Hm ... I'll see what I can do about Godot's limitations here, in case we decide to enforce read only in the future :)

Reply:
No that's fine, we pass in three arguments:

- Exercise slug
- Solution dir (input)
- Output dir

Feel free to write anything you want to the output dir. The `/tmp` dir is also writeable.

Reply:
OK, I've added `--read-only` back, but instead I created 3 volumes for directories that Godot needs to work properly (`.config`, `.local`, and `.cache`). If you're OK with this solution, I will gladly keep it ;)

Reply:
I'm sorry, but that won't work, as all test runners are mounted in the same way (so without those three new files).

Reply:
Hm ... I am not sure, but Godot might work without these folders, it will just print a lot of error messages ... I will check this. As long as the "results.json" file is produced correctly, we should be good, right?

Reply:
That is correct!

Reply:
OK, I've removed extra volumes and I'm redirecting stderr to a file in `/tmp`. If there is time later, I might add a filtering of those errors, so that the expected ones are ignored and the remaining ones are printed.